<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Results</class>
 <widget class="QWidget" name="Results">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Results of tests:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>530</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(224, 224, 224);
    border-radius: 5px;
    border: 2px solid rgb(184, 184, 184);
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(200, 200, 200);
    border: 2px solid rgb(154, 154, 154);
}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>781</width>
     <height>451</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>779</width>
      <height>449</height>
     </rect>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>781</width>
       <height>451</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <column>
      <property name="text">
       <string>Test Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Score</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Elapsed Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Result</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="graphical_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(224, 224, 224);
    border-radius: 5px;
    border: 2px solid rgb(184, 184, 184);
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(200, 200, 200);
    border: 2px solid rgb(154, 154, 154);
}</string>
   </property>
   <property name="text">
    <string>Graphical Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="export_btn">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>530</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(224, 224, 224);
    border-radius: 5px;
    border: 2px solid rgb(184, 184, 184);
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(200, 200, 200);
    border: 2px solid rgb(154, 154, 154);
}</string>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
